recipes/bamhash/1.1/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+export LDFLAGS=-L$PREFIX/lib
+export CXXFLAGS=-I$PREFIX/include
+
+make all
+
+mv bamhash_checksum_bam $PREFIX/bin
+mv bamhash_checksum_fastq $PREFIX/bin
```

Comment:
I don't think it gets compiled. There is no trace of it in the folders generated by conda-build.